### PR TITLE
{lang}[foss/2019b] CVXPY v1.0.28 w/ Python 3.7.4

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node376.hydra.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/25b0b7c35a4080afc2b1cba28819a156 for a full test report.

File: easybuild/easyconfigs/c/CVXPY/CVXPY-1.0.28-foss-2019b-Python-3.7.4.eb [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        'checksums': ['f816cfe3d4b4cff3ac2b8b96588c5960ddd2a3dc946bda6b09db04e7bc6577f2'],
+    }),
+    ('cvxpy', version, {
+        'checksums': ['212cd483e9a0fbc91d593aa599eb065615b058189f66c7881521b63e6493307a'],
+    }),
+]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages/%(namelower)s'],
```
Comment (boegel) [inline easybuild/easyconfigs/c/CVXPY/CVXPY-1.0.28-foss-2019b-Python-3.7.4.eb]: @smoors This is the default for `PythonBundle`,  so you can remove this custom `sanity_check_paths`

File: easybuild/easyconfigs/c/CVXPY/CVXPY-1.0.28-foss-2019b-Python-3.7.4.eb [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    ('Python', '3.7.4'),
+    ('SciPy-bundle', '2019.10', versionsuffix),
+    ('SWIG', '4.0.1'),
+]
+
+use_pip = True
+
+exts_default_options = {
+    'source_urls': [PYPI_SOURCE],
+    'sanity_pip_check': True,
```
Comment (boegel) [inline easybuild/easyconfigs/c/CVXPY/CVXPY-1.0.28-foss-2019b-Python-3.7.4.eb]: Style nitpicking: we usually set `sanity_pip_check` top-level, so:

```python
exts_default_options = {'source_urls': [PYPI_SOURCE]}

exts_list = [
    ...
]

sanity_check_paths = {
    ...
}

sanity_pip_check = True
```

File: easybuild/easyconfigs/c/CVXPY/CVXPY-1.0.28-foss-2019b-Python-3.7.4.eb (hunk 1 above)
Comment (smoors) [reply easybuild/easyconfigs/c/CVXPY/CVXPY-1.0.28-foss-2019b-Python-3.7.4.eb]: not exactly (I've added %(namelower)s), but fine for me.

File: easybuild/easyconfigs/c/CVXPY/CVXPY-1.0.28-foss-2019b-Python-3.7.4.eb [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+builddependencies = [
+    ('CMake', '3.15.3'),
+]
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('SciPy-bundle', '2019.10', versionsuffix),
+    ('SWIG', '4.0.1'),
```
Comment (Micket) [inline easybuild/easyconfigs/c/CVXPY/CVXPY-1.0.28-foss-2019b-Python-3.7.4.eb]: Isnt this just a builddep?

Comment (smoors) [reply easybuild/easyconfigs/c/CVXPY/CVXPY-1.0.28-foss-2019b-Python-3.7.4.eb]: yes, you're right.

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
hebbe-c1 - Linux centos linux 7.7.1908, Intel Core Processor (Haswell, no TSX), Python 2.7.5
See https://gist.github.com/904d0c414adcc824d69450effebcc4b1 for a full test report.

**Comment by Micket** [comment]
Going in, thanks @smoors!

**Pull request merged**